SpreadCheetah/Validations/DataValidation.cs

```diff
@@ -74,6 +75,50 @@ public ValidationErrorType ErrorType
 
     private ValidationErrorType _errorType;
 
+    #region DateTime validations
+    private static DataValidation _DateTime(ValidationOperator op, DateTime min, DateTime max)
+    {
+        if (max < min)
+            throw new ArgumentException(MinGreaterThanMaxMessage, nameof(min));
+        else
+        {
+            var epoch = new DateTime(1900,1,1,1,1,1,DateTimeKind.Unspecified);
+            var minElapsedDates = (min.AddDays(2) - epoch).Days;
+            var maxElapsedDates = (max.AddDays(2) - epoch).Days;
+
+            return new DataValidation(ValidationType.DateTime, op, minElapsedDates.ToString(CultureInfo.InvariantCulture), maxElapsedDates.ToString(CultureInfo.InvariantCulture));
+        }
+    }
+
+    private static DataValidation _DateTime(ValidationOperator op, DateTime value)
+    {
+        var epoch = new DateTime(1900,1,1,1,1,1,DateTimeKind.Unspecified);
```

Comment:
Maybe we can use `DateTime.ToOADate`? https://learn.microsoft.com/en-us/dotnet/api/system.datetime.tooadate

Reply:
We can certainly use OADate format. Bu what would we gain by using that format?

Reply:
You have almost implemented what `DateTime.ToOADate()` does, so you can replace this:
```csharp
var epoch = new DateTime(1900,1,1,1,1,1,DateTimeKind.Unspecified);
var elapsedDate = value.AddDays(2) - epoch;
var days = (elapsedDate.Days);
```
with
```csharp
var days = value.ToOADate();
```